docs/workflow/analyze.md

```diff
@@ -1,20 +1,140 @@
+
+*Correspondence model inspectation by scrolling though the dataset and zooming in & out*
+<p><video src="https://sci.utah.edu/~shapeworks/doc-resources/mp4s/studio_scroll.mp4" autoplay muted loop controls style="width:100%"></p>
+
+
+## Surface Reconstruction 
+
+For visualization purposes, the shape's particle system is used to reconstruct its surface mesh using a template-deformation approach to establish an inter-sample dense surface correspondence given a sparse set of optimized particles. To avoid introducing bias due to template choice, we use an unbiased framework for template mesh construction that entails:
+
+ - A generalized Procrustes alignment to define the mean particle system
+- A nonlinear warping function built using the shape's particle system and the mean one as control points.
+-  A mean distance transform (DT) computed by averaging warped sample-specific DT. 
+
+The template mesh will then be constructed by triangulating the isosurface of this mean DT. A warping function is constructed to deform the dense template mesh to the sample space using the sample's and mean particle systems as control points to reconstruct a sample-specific surface mesh.
+
+
+*Particle-based surface reconstruction currently supported by ShapeWorksStudio*
+![Template-deformation approach for surface reconstruction](../img/workflow/dt_based_recons.png)
+
+
+*(old) VTK-based surface reconstruction*
+<p><video src="https://sci.utah.edu/~shapeworks/doc-resources/mp4s/vtk_reconstruction.mp4" autoplay muted loop controls style="width:100%"></p>
+
+*(new) particle-based surface reconstruction*
+<p><video src="https://sci.utah.edu/~shapeworks/doc-resources/mp4s/particles_reconstruction.mp4" autoplay muted loop controls style="width:100%"></p>
+
+
+You can export any mesh using `File -> Export Current Mesh ...`. 
+
+
+## Statistical Analysis 
+
+Statistical analysis is performed using principal component analysis (PCA), where the mean and modes of shape variation are computed based on the optimized correspondence model. Automated animation is allowed to watch the shape morph at various standard deviations along a user-selected variation mode. 
+
+*Animating the shape variation along with a principal component while passing through the mean shape*
+
+<p><video src="https://sci.utah.edu/~shapeworks/doc-resources/mp4s/femur_first_mode.mp4" autoplay muted loop controls style="width:100%"></p>
+
+
+You can also export the eigenvalues and each shape's PCA loadings in an excel sheet for further analysis. 
+
+## Dynamic Loading
+
+ShapeWorksStudio support load-on-demand, which makes it more scalable by supporting loading in order of 100s of samples. For instance, an old small project that took 10+ seconds to load now starts instantly in Studio.
+
+
+*(old) without dynamic loading*
```

Comment:
Its nice to see these improvements, but is it relevant for a user interested in "how to analyze your shape model"?

Reply:
We could have a "What's new" section where we demonstrate new/improved features. other thoughts as what is the best way (and where) to "advertise" improved features? @akenmorris?

Reply:
I like the idea of a "whats new" section. Here's how the Godot project does it:

news: https://godotengine.org/news

docs: https://docs.godotengine.org/en/stable/

Reply:
Would the release notes have a summary for each release, but "what's new" section illustrate new/improve features in more detail?

Reply:
Done